dspace-server-webapp/src/test/java/org/dspace/app/rest/BitstreamRestRepositoryIT.java

```diff
@@ -1143,5 +1146,133 @@ public void testHiddenMetadataForUserWithWriteRights() throws Exception {
 
     }
 
+    @Test
+    public void getEmbeddedBundleForBitstream() throws Exception {
+        //We turn off the authorization system in order to create the structure as defined below
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and one collection.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                .withName("Sub Community")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        //2. One public items that is readable by Anonymous
+        Item publicItem1 = ItemBuilder.createItem(context, col1)
+                .withTitle("Test")
+                .withIssueDate("2010-10-17")
+                .withAuthor("Smith, Donald")
+                .withSubject("ExtraEntry")
+                .build();
+
+        String bitstreamContent = "ThisIsSomeDummyText";
+
+        //Add a bitstream to an item
+        Bitstream bitstream = null;
+        try (InputStream is = IOUtils.toInputStream(bitstreamContent, CharEncoding.UTF_8)) {
+            bitstream = BitstreamBuilder.
+                    createBitstream(context, publicItem1, is)
+                    .withName("Bitstream")
+                    .withDescription("Description")
+                    .withMimeType("text/plain")
+                    .build();
+        }
+
+        Bundle bundle = bitstream.getBundles().get(0);
+
+        //Get the bitstream with embedded bundle
+        getClient().perform(get("/api/core/bitstreams/" + bitstream.getID() + "?embed=bundle"))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(contentType))
+                .andExpect(jsonPath("$._embedded.bundle",
+                        BundleMatcher.matchProperties(
+                                bundle.getName(),
+                                bundle.getID(),
+                                bundle.getHandle(),
+                                bundle.getType()
+                        )
+                ));
+    }
+
+    @Test
+    public void linksToFirstBundleWhenMultipleBundles() throws Exception {
```

Comment:
I would prefer to remove this test as this is an use case that we don't want to support

Reply:
I'm OK with this test, as it proves that we are only ever returning the *FIRST* bundle if somehow an Item did exist in multiple bundles.

That said, I think @abollini is correct that we need to add a special comment on this method.  I'd recommend we add JavaDocs or a comment above this test method to say:
`This test proves that, if a bitstream is linked to multiple bundles, we only ever return the first bundle.  **NOTE: DSpace does NOT support or expect to have a bitstream linked to multiple bundles**.  But, because the database does allow for it, this test simply proves the REST API will respond without an error.`